[2024-10-24 18:01:56,532] 26 root - INFO - Data Ingestion Start
[2024-10-24 18:01:56,611] 31 root - INFO - I have read the dataset
[2024-10-24 18:01:56,757] 36 root - INFO - Successfully saved the raw data in the Artifacts folder
[2024-10-24 18:01:56,757] 39 root - INFO - Performing train-test split
[2024-10-24 18:01:56,764] 41 root - INFO - Train-test split completed
[2024-10-24 18:01:56,951] 47 root - INFO - Successfully saved the train and test data in the Artifacts folder
[2024-10-24 18:01:56,952] 48 root - INFO - Data Ingestion Complete
[2024-10-24 18:01:57,916] 136 root - ERROR - Error occurred during data transformation
[2024-10-24 18:01:57,916] 26 root - ERROR - Custom Exception: Error occurred in Python script name [e:\data_science _master\ml\shipment_price_prediction\src\ShipmentPricePrediction\components\data_transformation.py] line number [103] error message [could not convert string to float: 'See DN-3402 (ID#:83735)']
